src/Components/Alert/Components/Filters/ArtistSeries.tsx

```diff
@@ -0,0 +1,100 @@
+import { FC, Suspense } from "react"
+import { QuickMultipleSelectAlertFilter } from "Components/Alert/Components/Filters/QuickMultipleSelectAlertFilter"
+import { ArtistSeriesOptionsQuery } from "__generated__/ArtistSeriesOptionsQuery.graphql"
+import { graphql, useLazyLoadQuery } from "react-relay"
+import { useAlertContext } from "Components/Alert/Hooks/useAlertContext"
+import {
+  Flex,
+  Separator,
+  Skeleton,
+  SkeletonBox,
+  SkeletonText,
+} from "@artsy/palette"
+import { compact, times } from "lodash"
+
+const artistSeriesOptionsQuery = graphql`
+  query ArtistSeriesOptionsQuery($input: FilterArtworksInput!) {
+    artworksConnection(size: 0, input: $input) {
+      aggregations {
+        counts {
+          count
+          name
+          value
+        }
+        slice
+      }
+    }
+  }
+`
+
+export const ArtistSeries: FC = () => {
+  const { state } = useAlertContext()
+  const artistIDs = state?.criteria?.artistIDs
+
+  const data = useLazyLoadQuery<ArtistSeriesOptionsQuery>(
```

Comment:
We should use `useClientQuery` as that more adaptable to our suspense setup

Reply:
Do you mean the [custom hook](https://github.com/artsy/force/blob/main/src/Utils/Hooks/useClientQuery.ts) we have (vs the [one from Relay](https://relay.dev/docs/next/api-reference/use-client-query/)).

Is there any documentation about this? New to me, tbh…

Reply:
Yep, we use it all around in volt 2 and forque and a few places in force. It was added a while ago and fixes a few issues we've seen, not only with SSR (hydration errors, even tho its only supposed to render on client) but also the general UI inflexibility of suspense, due to the lack of a proper loading state (think, search drop down, etc). 

In volt 2 we added this to eslint to warn against use: https://github.com/artsy/volt-v2/blob/main/.eslintrc.js#L55-L59.